### ISSUE TITLE
Questions of transfer learning to validation images of LVIS v1.0

### ISSUE THREAD

**Issue by wzmw-zr** [scuwyh2000/RandBox#3]
In the paper, you state that you evaluate models trained on Task 1 on the 4,810 validation images of LVIS v1.0,  however, there are 19809 images in the validation set of LVIS v1.0. Will you release the images list used for transfer learning?

**Comment by scuwyh2000** [comment]
https://github.com/scuwyh2000/RandBox/releases/download/Tags/lvis-val.txt

**Comment by wzmw-zr** [comment]
Thank you! Sorry that I didn't notice the release files.

**Comment by Alice-302** [comment]
@wzmw-zr Is there a test.json file for LVIS v1.0?
